README.md

```diff
@@ -74,8 +89,8 @@ Housekeeping comes with a default `HousekeepingService` implementation, however
 
     @Bean
     HousekeepingService housekeepingService(
-        LegacyReplicaPathRepository legacyReplicaPathRepository) {
-      return new FileSystemHousekeepingService(legacyReplicaPathRepository, new org.apache.hadoop.conf.Configuration());
+        LegacyReplicaPathRepository legacyReplicaPathRepository, HouseKeeping houseKeeping) {
```

Comment:
Is `HouseKeeping` supposed to be camel cased like this?

Reply:
well spotted, it shouldn't no!